front/app/modules/commercial/content_builder/admin/components/RenderNode/index.tsx

```diff
@@ -136,7 +80,7 @@ const RenderNode = ({ render }) => {
   useEffect(() => {
     if (
       isActive &&
-      name === CONTAINER &&
+      name === 'Container' &&
```

Comment:
Minor: It might be better to make 'Container' a constant. As it is, it is easier for one usage to change without the others which would be undesired.